compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -49,6 +49,7 @@ class CompilationTests extends ParallelTesting {
       compileFilesInDir("tests/new", defaultOptions),
       compileFilesInDir("tests/pos-scala2", scala2CompatMode),
       compileFilesInDir("tests/pos-custom-args/erased", defaultOptions.and("-Yerased-terms")),
+      compileFilesInDir("tests/pos-custom-args/semanticdb", defaultOptions.and("-Ysemanticdb")),
```

Comment:
If all tests are run with Ysemantic-db then that option isn't needed right?

Reply:
I would say keep this in case a decision is made to remove -Ysemanticdb from checkOptions, these tests are specifically designed for regression testing the implementation